README.md

```diff
@@ -109,35 +108,34 @@ Search links that point directly to suitable issues to contribute to on GitHub.
 - [Try Git](https://try.github.io/) - Learn Git in 15 minutes from within your browser for free.
 - [Everyday Git](https://git-scm.com/docs/giteveryday) - A useful minimum set of commands for Everyday Git.
 - [Oh shit, git!](https://ohshitgit.com/) - how to get out of common `git` mistakes described in plain English; also see [Dangit, git!](https://dangitgit.com/) for the page without swears.
-- [Atlassian Git Tutorials](https://www.atlassian.com/git/tutorials/) - various tutorials on using `git`.
+- [Atlassian Git Tutorials](https://www.atlassian.com/git/tutorials) - various tutorials on using `git`.
 - [GitHub Git Cheat Sheet](https://education.github.com/git-cheat-sheet-education.pdf) (PDF)
-- [freeCodeCamp's Wiki on Git Resources](https://www.freecodecamp.org/forum/t/wiki-git-resources/13136)
+- [freeCodeCamp's Wiki on Git Resources](https://forum.freecodecamp.org/t/wiki-git-resources/13136)
 - [GitHub Flow](https://www.youtube.com/watch?v=juLIxo42A_s) (42:06) - GitHub talk on how to make a pull request.
-- [GitHub Learning Resources](https://help.github.com/articles/git-and-github-learning-resources/) - Git and GitHub learning resources.
+- [GitHub Learning Resources](https://docs.github.com/articles/git-and-github-learning-resources/) - Git and GitHub learning resources.
```

Comment:
Weirdly enough, `awesome_bot` **doesn't** want a forward slash at the end.

```suggestion
- [GitHub Learning Resources](https://docs.github.com/articles/git-and-github-learning-resources) - Git and GitHub learning resources.
```

https://github.com/freeCodeCamp/how-to-contribute-to-open-source/pull/433/checks?check_run_id=1201626133